core/openapi/parameters/language.yaml

```diff
@@ -0,0 +1,12 @@
+name: language
```

Comment:
I thought you would go for a relatively loose "note" as in Features for the accept header, e.g. 

> Requirement X implies that the language of a server response is determined using content negotiation as specified by the HTTP RFC.
> 
> Note that any server that supports multiple languages will have to support a mechanism to mint encoding-specific URIs for resources in order to express links, for example, to alternate representations of the same resource. This document does not mandate any particular approach how this is supported by the server.
> 
> As clients simply need to dereference the URI of the link, the implementation details and the mechanism how the language is included in the URI of the link are not important. Developers interested in the approach of a particular implementation, for example, to manipulate ("hack") URIs in the browser address bar, can study the API definition.
>
> NOTE: A common approach is an additional query parameter (for example, "language" or "accept-language") that overrides the Accept-Language header of the HTTP request.

I'm fine either way though, we can also go for a specific `language` query parameter, but in this case the schema is not ideal I think. In #195 you asked that the parameter is aligned with the accept-language header. Although the schema mandates comma separated values, it's not clear from the schema that it can include the q thing etc. I assume it could be something simple like:

```yml
  type: array
  items:
    type: string
    pattern: "^([a-z]+)(-[a-zA-Z0-9]+)+(;q=(0|0.\d+|1|1.0+))?$"
```

If you don't go for the pattern, a description how this needs to look like (i.e. equivalent to the accept-language header) would be good.

Reply:
@m-mohr I have updated the `langauge` parameter defined in `language.yaml` to match STAC.  Please review and let me know if it is better now.

I should point out that the definition of this parameter is not normative as the specification does not mandate a particular apprach for minting language specific URLs.  I realized, however, that I was not referencing that prameter schema (i.e. language.yaml) anywhere in the text of the specification so I now include it the the section titled "Langauge of the record" as an example of how a server can mint language specific URLs.

As for your loose note, the API for records is the Feature API so this is language that the features API should  include somehwere.  All records says, with regard to the API, is do what Features does.  Specifically, records says:

Implementations of the Records API SHOULD comply with recommendation [`/req/core/string-i18n`](http://docs.ogc.org/is/17-069r3/17-069r3.html#string_i18n) of the [OGC API - Features - Part 1: Core](http://docs.ogc.org/is/17-069r3/17-069r3.html) standard.

Reply:
Looks good, thanks.